.editorconfig

```diff
@@ -10,5 +10,5 @@ trim_trailing_whitespace = true
 [*.md]
 trim_trailing_whitespace = false
 
-[{*.json,.eslintrc,.babelrc}]
-indent_size = 2
\ No newline at end of file
+[*.json]
```

Comment:
So we're treating .eslintrc different from other json files? (as per next line, the indent for json is 4)